pidtree_bcc/plugins/identityplugin.py

```diff
@@ -0,0 +1,8 @@
+from pidtree_bcc.plugin import BasePlugin
+
+class Identityplugin(BasePlugin):
+    def __init__(self, args):
+        super
```

Comment:
`super` must be called with args, or just remove this `__init__` since nothing is done in it

Reply:
Ack